src/utils/sformat.fs

```diff
@@ -916,9 +918,9 @@ namespace Microsoft.FSharp.Text.StructuredFormat
             match c with 
```

Comment:
Am I right in saying that F# files are parsed as UTF-8? In which case I don't think we need to escape anything else other than what's there already.